Source/Scene/Material.js

```diff
@@ -853,13 +854,29 @@ function createTexture2DUpdateFunction(uniformId) {
       return;
     }
 
+    if (uniformChanged && defined(texture)) {
+      if (
+        texture !== this.defaultTexture &&
+        texture !== context.defaultTexture
```

Comment:
`this.defaultTexture` - is this supposed to be `material._defaultTexture`?

Can `texture !== context.defaultTexture` be removed since it's the same thing?

Maybe `material._defaultTexture = context.defaultTexture` can be set as one of the first lines in `Material.prototype.update`. I'm starting to lose track of when `_defaultTexture` is set and when it's not set. Really this is a problem with the existing code but it's worth cleaning up.